### ISSUE TITLE
Investigate if there is better options for inch and foot marks

### ISSUE THREAD

**Issue by bruce-usab** [atbcb/usab-uswds#1529]
Semantically, using a neutral double or single quote mark for the inch and foot mark symbol is not _quite_ correct.

@bruce-usab thinks it is the best we can do.  @kengdoj thinks we can do better!

Edit to add link to [article on foot and inch marks](https://practicaltypography.com/foot-and-inch-marks.html) from [Typography for Lawyers](https://typographyforlawyers.com/).  This site has several good articles, including one on [straight and curly quotes](https://typographyforlawyers.com/straight-and-curly-quotes.html).  Since it mentions lawyers, @tc-usab will like it!

**Comment by kengdoj** [comment]
Ex: 24"-36" 

- My NVDA testing did not announce the double quotes and apostrophe marks used to indicate inch and feet, and announced "minus" for hyphen. NVDA reads the example as "24 minus 36".
- Use of aria-label to announce "inch", "feet", and "to" instead of their text characters did not work with NVDA.

NVDA did work using the HTML code for inch and feet, so
- HTML codes for inch (double prime ″) and feet (prime ′) should replace the double quote and apostrophe marks.
- The word "to" should replace '-'.

**Comment by bruce-usab** [comment]
So many questions I have!  I am not enthusiastic about a conclusion that (1) is more work and (2) makes things worse for JAWS users.

- Does NVDA have verbosity setting?
- What does NVDA do with prime and double prime when used for minute/seconds (as in latitude / longitude )?
- Please test minus, hyphen, and en dash.  Those are three different HTML characters.  If NVDA is pronouncing hyphen as minus, that seems like a bug.
- JAWS knows that words like wind and read can be pronounced two ways, and uses context to pick the correct one most of the time.
- - Does NVDA have similar powers?
- - Can this context mechanism be exploited to have JAWS (and maybe NVDA) do between with these common symbols?

I think we need a bigger user sample.  WebAIM does an annual SR survey, so they might be a good source.

**Comment by kengdoj** [comment]
Here's a solution that I think will work:
- replace the quote/apostrophe with double prime/prime for inch/feet. Hide from AT. Inch/feet text is visually hidden but announced correctly.
- replace hyphen with ndash for "to". Hide from AT. "to" text is visually hidden but announced correctly.

```
<style>
.sr-only {
      border: 0; 
      clip: rect(0 0 0 0); 
      height: 1px; 
      margin: -1px;
      overflow: hidden;
      padding: 0;
      position: absolute;
      width: 1px;
    }
</style>
```

| Search | Replace |
|--------|----------|
| ([0-9])" to | $1`<span aria-hidden="true">&#8243; </span> <span class="sr-only">inches to</span> <span aria-hidden="true">&ndash;</span>` |
| ([0-9])' to | $1`<span aria-hidden="true">&#8242; </span> <span class="sr-only">feet to</span> <span aria-hidden="true">&ndash;</span>` |

(@PresleyAB  approves.)

**Comment by bruce-usab** [comment]
Bruce is of the opinion that this solution is more harmful than the problem it purports to address.

It may come down to the WCAG definition for [non-text content](https://www.w3.org/TR/WCAG22/#dfn-non-text-content).  Bruce asserts that most typographical marks, including &39; and &34; are programmatically determinable and that the sequence (a number followed by a single particular character, \&39;) _is_ expressing something in human language.

Using double prime &Prime; would be better from a semantic perspective, but that character is not well-supported by screen reading software.  Using <quot> conveys the semantic intent to all users (well, all users familiar with imperial units) and is a sufficient approach.

**Comment by kengdoj** [comment]
A correct fix for this is more annoying for screen readers:

`
24<span role="img" aria-label="inch">"</span>
`

Here, "image inch" would be announced every time, regardless of the visual symbol for inch.

**Comment by bruce-usab** [comment]
IMHO <q>more annoying</q> is not compatible with <q>correct</q>.  Which is why I am not much interested in using \&Doubleprime;

One approach I plan to explore is using ABBR, e.g., `\<abbr class="sr-hint">title=\"inch\">\"</abbr>` 

Semantically that is minimally acceptable, and we might even be able (at some point) to swap out ASCI 34 for UniCode 2033

**Comment by kengdoj** [comment]
> `<abbr class="sr-hint" title="inch">"</abbr>`

Interesting. Very interesting. I'd be interested in whether screen readers announce the symbol.

**Comment by kengdoj** [comment]
Resolution for Guide pages: (anywhere else?)

1. Use double prime for inch, prime for feet and 
2. Add sr tip at the top of the ADA Guide pages about what these symbols represent.

**Comment by bruce-usab** [comment]
That was my take-away.  But also:

0. Write out inch (in letters) where it is practical to do so.  Also spell out words like minimum and maximum where practical (instead of using "min." and "max.").

For Markdown `&Prime;` and `&prime;` work.  One might also be able to copy/paste and type directly:  `Alt+8243` (digits from numeric keypad).

**Comment by kengdoj** [comment]
Closed by #1603
